doc/examples/tissue_classification.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Apr 20 15:57:42 2016
+
+@author: jvillalo
+"""
+"""
+=======================================================
+Tissue Classification of a T1-weighted Strutural Image
+=======================================================
+This example explains how to segment a T1-weighted structural image using a MRF
+approach. Similar algorithms have been proposed by Zhang et al. [Zhang2001]_,
+and Avants et al. [Avants2011]_ available in FAST-FSL and ANTS-Artropos,
+respectively.
+
+Here we will use a T1-weighted image, that has been previously skull-stripped
+and bias field corrected.
+"""
+
+import numpy as np
+#import nibabel as nib
+import matplotlib.pyplot as plt
+from dipy.denoise.nlmeans import nlmeans
+from dipy.denoise.noise_estimate import estimate_sigma
+from dipy.data import fetch_tissue_data, read_tissue_data
+from dipy.segment.tissue import TissueClassifierHMRF
+
+"""
+First we fetch the T1 volume from the Syn dataset and will determine its shape.
+"""
+
+fetch_tissue_data()
+t1_img = read_tissue_data()
+t1 = t1_img.get_data()
+print('t1.shape (%d, %d, %d)' % t1.shape)
+
+"""
+Once we have fetched the T1 volume, we proceed to denoise it with the NLM algorithm.
```

Comment:
Ok, yes I agree. Got rid of the denoising of the T1. It makes the example complicated and made the run time longer.